docs/integrations-guides/rollup-guides/orbit/overview.md

```diff
@@ -44,6 +82,7 @@ Many modifications were made to successfully support fraud proofs and secure bat
 
 - [Extended](https://github.com/Layr-Labs/nitro/blob/e8981ff2f09720b6627e751d8bd3146277c7a01b/arbnode/batch_poster.go#L113) batch poster to take in an `eigenDAWriter` struct that writes blobs to DA via eigenda-proxy
     - [Embed](https://github.com/Layr-Labs/nitro/blob/e8981ff2f09720b6627e751d8bd3146277c7a01b/arbnode/batch_poster.go#L1030-L1075) ABI calldata for tx submissions to `SequencerInbox`
+    - [Provide](https://github.com/Layr-Labs/nitro/pull/46) programmatic failover feature to use other DA destinations when EigenDA is unavailable.
 - [Extended](https://github.com/Layr-Labs/nitro/blob/e8981ff2f09720b6627e751d8bd3146277c7a01b/arbnode/sequencer_inbox.go#L174-L211) inbox message derivation to support type processing for an eigenda batch type (i.e, `0xed` prefix)
```

Comment:
*EigenDA

Reply:
` programmatic failover feature to use other DA destinations when EigenDA is unavailable.` ?